app/gui/src/project-view/components/ComponentBrowser.vue

```diff
@@ -199,7 +201,9 @@ watchEffect(() => {
     return
   }
   const scenePos = originScenePos.value.add(
-    new Vec2(COMPONENT_EDITOR_PADDING + ICON_WIDTH / 2, 0).scale(clientToSceneFactor.value),
+    new Vec2(COMPONENT_EDITOR_PADDING + ICON_WIDTH / 2, 0)
+      .scale(clientToSceneFactor.value)
+      .add(new Vec2(0, EDGE_Y_OFFSET)),
```

Comment:
Does this look right at extreme zoom levels? I would think the `EDGE_Y_OFFSET` should have a constant size in *client* pixels, in which case it should be added before scaling.